packages/core/src/new-api/journal.ts

```
@@ -0,0 +1,45 @@
+import { IgnitionError } from "../errors";
+
+import { Journal, JournalableMessage } from "./types/journal";
+
+/**
+ * An in-memory journal.
+ *
+ * @beta
+ */
+export class MemoryJournal implements Journal {
+  private messages: string[] = [];
+
+  public async record(message: JournalableMessage): Promise<void> {
+    try {
+      this.messages.push(JSON.stringify(message));
+    } catch (err) {
+      console.error(err, message);
```

I guess this was here for debugging purposes, but it's a (semi)silent error.